src/Native/System.Native/pal_process.h

```diff
@@ -16,49 +16,45 @@
  * As would have been the case with fork/execve, a return value of 0 is success and -1
  * is failure; if failure, error information is provided in errno.
  */
-extern "C"
-int32_t ForkAndExecProcess(
-    const char* filename,    // filename argument to execve
-    char* const argv[],      // argv argument to execve
-    char* const envp[],      // envp argument to execve
-    const char* cwd,         // path passed to chdir in child process
-    int32_t redirectStdin,    // whether to redirect standard input from the parent
-    int32_t redirectStdout,   // whether to redirect standard output to the parent
-    int32_t redirectStderr,   // whether to redirect standard error to the parent
-    int32_t* childPid,        // [out] the child process' id
-    int32_t* stdinFd,         // [out] if redirectStdin, the parent's fd for the child's stdin
-    int32_t* stdoutFd,        // [out] if redirectStdout, the parent's fd for the child's stdout
-    int32_t* stderrFd);       // [out] if redirectStderr, the parent's fd for the child's stderr
-
+extern "C" int32_t ForkAndExecProcess(const char* filename,   // filename argument to execve
+                                      char* const argv[],     // argv argument to execve
```

Comment:
Is there an option to put parameters as they were above when on more than one line. It saves a lot of horizontal real estate and causes less churn if function is renamed?